### PR TITLE
Convert selected blog posts and transition email

### PR DESCRIPTION
**Pull request by michelle-rago**
Add the full text of blog posts that we previously just showed summaries for.

- Add ".gov registrar transition update"
- Add transition email
- Add "infrastructure as a public service"

**[Review content here](https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/get.gov/convert-selected-blog-posts/blog/)**

### CODE REVIEW

File: posts/2023-10-13-registrar-transition-update.md [hunk 1]
```diff
@@ -9,4 +9,76 @@ tags:
...
+
+- - -
+
+## What we heard from you
+
+Thank you for your questions and comments! Here are responses to what we heard.
+
+* [Does this mean that all .gov websites will shut down for three months?](#does-this-mean-that-all-gov-websites-will-shut-down-for-three-months)
+* [Can we launch new subdomains on .gov sites?](#can-we-launch-new-subdomains-on-gov-sites)
+* [What if some .gov domain managers don't create Login.gov accounts?](#what-if-some-gov-domain-managers-dont-create-logingov-accounts)
```
Comment (h-m-f-t) [inline posts/2023-10-13-registrar-transition-update.md:64]: The anchors changed in 11ty (the ID includes the question mark and percent encodes it `%3F`). Can you take a quick look at these?

Comment (michelle-rago) [reply posts/2023-10-13-registrar-transition-update.md]: Fixed; sorry about that!

File: posts/2023-09-06-coming-soon-gov-changes.md [hunk 2]
```diff
@@ -9,4 +9,32 @@ tags:
...
+
+2. **A new online home for .gov domains**: We’re launching a new *registrar*, the online home for governments to request and manage a .gov domain. We put the domain request process online (no more physical signatures required). The new registrar will enable many future features, like DNS hosting. Moving to the new registrar requires our users to take some actions, which we detailed in an [email to domain registrants](../2023-transition-email/). 
+
+3. **A new website for .gov**: Finally, we’re sharing a *beta version of our new website*, which will help governments learn about .gov and why they should use it. We’re still building and testing [get.gov](https://get.gov/). 
+
+Both the registrar and our new website are being developed as open source software. You can check out the code for [the registrar](https://github.com/cisagov/getgov){.usa-link--external}  and our [public site](https://github.com/cisagov/get.gov){.usa-link--external} on GitHub.
```
Comment (h-m-f-t) [inline posts/2023-09-06-coming-soon-gov-changes.md:24]: Change first repo name to cisagov/manage.get.gov

**Review by h-m-f-t — APPROVED** [review]

File: posts/2023-09-06-coming-soon-gov-changes.md (hunk 2 above)
Comment (michelle-rago) [reply posts/2023-09-06-coming-soon-gov-changes.md]: Fixed. Thank you!

**Pull request merged**